src/tendril.rs

```diff
@@ -554,6 +554,21 @@ impl<F, A> Tendril<F, A>
         }
     }
 
+    /// Get the capacity of the `Tendril`.
+    ///
+    /// The capacity is the maximum number of underlyning items that can be added to this
+    /// `Tendril` before triggering an realocation.
+    #[inline(always)]
```

Comment:
Why `#[inline(always)]` rather than `#[inline]` here? I seem to remember the former was recommended against, but can’t find a reference at the moment.